composer.json

```diff
@@ -31,14 +31,18 @@
         "drush/drush": "^10.4"
     },
     "scripts": {
+        "coding-standards-check/twigcs": [
+            "twigcs web/profiles/custom/os2loop/themes/os2loop_theme/templates"
+        ],
         "post-install-cmd": [
             "ln -sf ../../scripts/git/hooks/pre-commit .git/hooks/pre-commit"
         ],
         "check-coding-standards/phpcs": [
             "vendor/bin/phpcs --standard=phpcs.xml.dist"
         ],
         "check-coding-standards": [
-            "@check-coding-standards/phpcs"
+            "@check-coding-standards/phpcs",
+            "@coding-standards-check/twigcs"
```

Comment:
For consistency this should be `check-coding-standards/twigcs`, but a general change to `coding-standards-check` (and `coding-standards-apply`) will also be approved.